src/pages/registration/contributors_tab/AddContributorModal.tsx

```diff
@@ -38,7 +38,6 @@ export const AddContributorModal = ({
   const handleCloseModal = () => {
     toggleModal();
     setOpenAddUnverifiedContributor(false);
-    setSearchTerm('');
```

Comment:
TODO: fjerning av dette gir en ny feil om man feks skal legge til 2 nye bidragsytere på rad 🤔

Reply:
seff, må sette verdien på samme måte som default for staten...